CONTRIBUTING.md

```diff
@@ -0,0 +1,95 @@
+## How to contribute code
+
+Follow these steps to submit your code contribution.
+
+### Step 1. Open an issue
+
+Before making any changes, we recommend opening an issue (if one doesn't already
+exist) and discussing your proposed changes. This way, we can give you feedback
+and validate the proposed changes.
+
+If the changes are minor (simple bug fix or documentation fix), then feel free
+to open a PR without discussion.
+
+### Step 2. Make code changes
+
+To make code changes, you need to fork the repository. You will need to setup a
+development environment and run the unit tests. This is covered in section
+"Setup environment".
+
+### Step 3. Create a pull request
+
+Once the change is ready, open a pull request from your branch in your fork to
+the master branch in [keras-team/keras](https://github.com/keras-team/keras).
+
+### Step 4. Sign the Contributor License Agreement
+
+After creating the pull request, the `google-cla` bot will comment on your pull
+request with instructions on signing the Contributor License Agreement (CLA) if
+you haven't done so. Please follow the instructions to sign the CLA. A `cla:yes`
+tag is then added to the pull request.
+
+![Tag added](https://i.imgur.com/LHEdIfL.png)
+
+
+### Step 5. Code review
+
+CI tests will automatically be run directly on your pull request.  Their
+status will be reported back via GitHub actions.
+
+There may be
+several rounds of comments and code changes before the pull request gets
+approved by the reviewer.
+
+![Approval from reviewer](https://i.imgur.com/zgRziTt.png)
+
+### Step 6. Merging
+
+Once the pull request is approved, a `ready to pull` tag will be added to the
```

Comment:
Since we are using github as source of truth, the 'ready to pull' is not used. The keras team member only need to merge the PR from github and doesn't need any copybara workflow to kickin.

Reply:
Good point - yeah I missed this.  Thanks